test/resources/ostree.resource

```diff
@@ -30,15 +23,15 @@ Current Deployment Should Be
 Deploy Commit Expecting A Rollback
     [Documentation]    Deploys given ref and configures test agent for failing greenboot.
     ...    It expects the system to roll back.
-    [Arguments]    ${ref}
+    [Arguments]    ${ref}    ${write_agent_cfg}=${TRUE}
 
     ${initial_deploy_id}=    Get Booted Deployment Id
     ${deploy_id}=    Rebase System    ${ref}
-    TestAgent.Write
-    Write Greenboot Microshift Wait Timeout    60
+    IF    ${write_agent_cfg}    TestAgent.Write
```

Comment:
Is the config always the same?

Reply:
No, this line is only persist what test should set up, see [fdo.robot#L33](https://github.com/openshift/microshift/blob/main/test/suites-ostree/fdo.robot#L33) and [failed-upgrade.robot#L30](https://github.com/openshift/microshift/blob/main/test/suites-ostree/failed-upgrade.robot#L30)

Reply:
Ah, right, I forgot the config was managed separately from the write step.